From 676935e57163fe87bec027dea9c12a5c3c41832f Mon Sep 17 00:00:00 2001
From: Michael Krummsdorf <michael.krummsdorf@tq-group.com>
Date: Wed, 17 Jan 2018 12:15:32 +0100
Subject: [PATCH 044/188] phy: dp83867: change reset depth

Change Reset from SW_RESET (including registers) to
SW_RESTART (not including registers).

Otherwise this resets the source of CLK_OUT in
IOMUX_CFG register, leading to a wrong frequency
on that line used as reference clock input.

Signed-off-by: Michael Krummsdorf <michael.krummsdorf@tq-group.com>
---
 drivers/net/phy/dp83867.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/phy/dp83867.c b/drivers/net/phy/dp83867.c
index 5cfe5dd21e06..057bdf1df219 100644
--- a/drivers/net/phy/dp83867.c
+++ b/drivers/net/phy/dp83867.c
@@ -322,7 +322,7 @@ static int dp83867_phy_reset(struct phy_device *phydev)
 {
 	int err;
 
-	err = phy_write(phydev, DP83867_CTRL, DP83867_SW_RESET);
+	err = phy_write(phydev, DP83867_CTRL, DP83867_SW_RESTART);
 	if (err < 0)
 		return err;
 
